doc/Makefile

```diff
@@ -48,6 +48,16 @@ html-noplot:
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html/stable."
 
+html-pattern:
+	# These two lines make the build a bit more lengthy, and the
```

Comment:
But the second is commented out

Reply:
Copied and pasted from `html` pattern (for better or worse) ...